sklearn/ensemble/_gb.py

```diff
@@ -165,7 +194,6 @@ def __init__(
         n_iter_no_change=None,
         tol=1e-4,
     ):
-
```

Comment:
Let's try to minimize empty alterations like this line deletion to preserve `git blame` history

Reply:
I didn't intentionally deleted these lines, black (v22.6.0) did! I will revert these changes.

Reply:
Ah gotcha. I think we support `black==22.3.0` so you may want to downgrade your version (fewer headaches 😉)